courtroom.cpp

```diff
@@ -1562,7 +1562,7 @@ void Courtroom::play_sfx()
   if (sfx_name == "1")
     return;
 
-  QVector<QString> extensions{"", ".ogg", ".wav", ".mp3"};
+  QVector<QString> extensions{"", ".wav", ".ogg", ".opus", ".mp3"};
```

Comment:
I see that single `wav` being out-of-order bothered ya. :P

Reply:
I mean may as well make it the same across the board. (+copy pasta works wonder)